solver/Cargo.toml

```diff
@@ -15,4 +15,6 @@ primitive-types = "0.7"
 reqwest = { version = "0.10", features = ["json"] }
 serde_json = "1.0"
 tokio = { version = "0.2", features =[ "macros", "time"] }
+tracing = "0.1"
+tracing-setup = { path = "../tracing-setup" }
```

Comment:
Does it make sense to re-export the tracing macros and just depend our own logging crate (this way the concrete logging library would be abstracted away). Or is there too much tracing specific things that would make this abstraction too leaky?

Reply:
I thought about this too but I think it makes sense for the other crates to use tracing directly. Tracing has a couple of improvements in the macros over log so to use those we wouldn't be compatible anyway.

Reply:
One possibility is to add to `tracing-setup/src/lib.rs`
```rust
pub use tracing;
```

Then the dependencies would use it like:
```rust
use tracing_setup::tracing;

tracing::info!("");
```

This just ensures all versions of `tracing` stay the same throughout the workspace.

(I would probably also `s/tracing-setup/something-else-i-have-no-idea-how-to-name-things/` in this case as well).

Reply:
/cc @e00E as this PR is already marked for merging.

Reply:
True that it trivially makes the version stay the same but has the disadvantage that the other crates have to `use` it first instead of doing `tracing::info!` directly which I like.

Reply:
> instead of doing tracing::info! directly which I like

I think that's a really good point especially for logging, as removing any barriers to being able to add logs is a good thing.